pkg/logstash.sysv

```diff
@@ -114,6 +114,27 @@ force_stop() {
   fi
 }
 
+config_test() {
+  # Check if a config file exists
+  if [ ! "$(ls -A $LS_CONF_DIR/*.conf 2> /dev/null)" ]; then
+    log_failure_msg "There aren't any configuration files in $LS_CONF_DIR"
+    exit 1
+  fi
+
+  JAVA_OPTS=${LS_JAVA_OPTS}
+  HOME=${LS_HOME}
+  export PATH HOME JAVA_OPTS LS_HEAP_SIZE LS_JAVA_OPTS LS_USE_GC_LOGGING
+
+  test_args="-f ${LS_CONF_DIR} --configtest ${LS_OPTS}"
+  if [ "$1" = '1' ]; then
+    $program ${test_args} 2>/dev/null >/dev/null
+  else
+    $program ${test_args}
+  fi
+  [ $? -eq 0 ] && return 0
+  # Program not configured
+  return 6
```

Comment:
program is not configured

Check out the bottom of the page http://refspecs.linuxbase.org/LSB_3.1.0/LSB-Core-generic/LSB-Core-generic/iniscrptact.html